examples/hetero/han_imdb.py

```diff
@@ -0,0 +1,89 @@
+from typing import Union, Dict, List
+import os.path as osp
+
+import torch
+from torch import nn
+import torch.nn.functional as F
+
+from torch_geometric.datasets import IMDB
+from torch_geometric.nn import HANConv
+import torch_geometric.transforms as T
+
+path = osp.join(osp.dirname(osp.realpath(__file__)), '../../data/IMDB')
+metapaths = [[('movie', 'actor'), ('actor', 'movie')],
+             [('movie', 'director'), ('director', 'movie')]]
+transform = T.AddMetaPaths(metapaths=metapaths, drop_orig_edges=True)
+dataset = IMDB(path, transform=transform)
+data = dataset[0]
+del data['director'], data['actor']
+print(data)
+
+
+class han(nn.Module):
+    def __init__(self, in_channels: Union[int, Dict[str, int]],
+                 out_channels: int, hidden_channels=128, heads=8):
+        super().__init__()
+        self.han_conv = HANConv(in_channels, hidden_channels, heads=heads,
```

Comment:
Does the paper only make use of a single layer as well?

Reply:
It looks like that, but the paper isn't entirely clear on that. 
Also the F1 score I am seeing is very similar to the one reported in the paper.

Reply:
Super :)